paddle/fluid/operators/fc_op.cc

```diff
@@ -110,6 +119,25 @@ void FCOpMaker::Make() {
   AddInput("W", "(Tensor), The weight fc op with shape (I, O).");
   AddInput("Bias", "(Tensor, optional) Bias vector with shape (1 x O")
       .AsDispensable();
+
+  AddAttr<float>("Scale_in",
+                 "(float, default 1.0f), The quantize scale of input data")
+      .SetDefault(1.0f);
+  AddAttr<std::vector<float>>("Scale_weights",
+                              "(std::vector<float>, default {1.0f}), The "
+                              "quantize scale of weights data")
+      .SetDefault({1.0f});
+  AddAttr<float>("Scale_out",
+                 "(float, default 1.0f), The quantize scale of output data")
+      .SetDefault(1.0f);
+  AddAttr<std::string>(
+      "data_format",
+      "(string, default NCHW) Only used in "
+      "An optional string from: \"NHWC\", \"NCHW\". "
+      "Defaults to \"NHWC\". Specify the data format of the output data, "
+      "the input will be transformed automatically. ")
+      .SetDefault("AnyLayout");
```

Comment:
Although this is a temp file, please take it much seriously for reviewers. fc doesn't need data_format.

Reply:
@luotao1 "data_format" was needed by FC INT8 kernel, please see [line150](https://github.com/PaddlePaddle/Paddle/pull/17097/files/edd7bd3ecce6bc9d619f2e415273b977abac9165#diff-cd3543641d6c976ea46111fc3dfab3f6R150) for it.